.gitignore

```diff
@@ -104,11 +100,6 @@ frontend/categories/*.yaml
 # Template for the local runtime configuration.
 !/config/local*.template.yml
 
-# Gatsby
-/frontend/.cache
-/frontend/public
-/public
```

Comment:
If you already have a checkout of shields, these removed paths will become unstaged files in your checkout after pulling this. Other than "just live with it", is there any good pattern for dealing with this?

Reply:
not that i know of. it's something i've experienced occasionally in a few projects, and always just dealt with via local cleanups (either via git or just standard file system management tooling).

i do not think this is worth the effort, especially given our current quantity of regular contributors, but the only potential automated/assistive route i can think of would be seeing if there's any git client hooks (post-merge maybe?) we could hook into (pun very much intended) that would deal with any such known files/directories for our project